extensions/cornerstone/src/Viewport/Overlays/CustomizableViewportOverlay.tsx

```diff
@@ -302,26 +306,41 @@ function CustomizableViewportOverlay({
   );
 }
 
-function _getViewportInstances(viewportData) {
+function _getViewportInstances(viewportData, displaySetService, instanceIndex) {
   const imageIds = [];
+  const displaySets = []
   if (viewportData.viewportType === Enums.ViewportType.STACK) {
```

Comment:
Replace the code which gets the imageIds with code that gets a display set UID - something like:
const { displaySetInstanceUID } = viewportData?..data?.[0] || {};
if( !displaySetInstanceUID ) return

then get the display set from the response, and return 
{
   displaySet,
   instance: displaySet.instances[instanceIndex],
   referenceInstance: displaySet.instance,
}
and just add that bit right into the properties.